grpc/proto/v1/search_get.proto

```diff
@@ -125,15 +125,15 @@ message Hybrid {
   string query = 1;
   repeated string properties = 2;
   // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
-  repeated float vector = 3 [deprecated = true];  // will be removed in the future, use vector_bytes
+  repeated float vector = 3 [deprecated = true];  // will be removed in the future, use vectors
   float alpha = 4;
   enum FusionType {
     FUSION_TYPE_UNSPECIFIED = 0;
     FUSION_TYPE_RANKED = 1;
     FUSION_TYPE_RELATIVE_SCORE = 2;
   }
   FusionType fusion_type = 5;
-  bytes vector_bytes = 6;
+  bytes vector_bytes = 6 [deprecated = true];  // will be removed in the future, use vectors
```

Comment:
could you add the version it was deprecated (1.29), so we can easily figure out when to remove it?